apps/v1/types.go

```diff
@@ -340,6 +340,10 @@ type DeploymentCauseImageTrigger struct {
 	From corev1.ObjectReference `json:"from" protobuf:"bytes,1,opt,name=from"`
 }
 
+type DeploymentStatus string
```

Comment:
we were usually keeping those close to the enum definition, is this a new pattern in opeshift/api?

Reply:
it is types vs. const I don't feel strong about where this definition will live

Reply:
No problem, I will move it